.goreleaser.yml

```diff
@@ -40,7 +40,5 @@ snapshot:
 changelog:
   sort: asc
   filters:
-    include:
-      - '^fix:'
-      - '^feat:'
-      - '^perf:'
+    exclude:
+      - '^(?!(?:fix|feat|perf):\s+)'
```

Comment:
icky but apparently goreleaser does not currently support `include:` and what i really want in changelogs are changes that generate releases. negate the exclude it is.